src_cpp/opennnBridge/nerlWorkerOpenNN.cpp

```diff
@@ -49,10 +51,17 @@ namespace nerlnet
         {
             case MODEL_TYPE_AE_CLASSIFIER:
             {
-                loss_val_tensor = std::make_shared<fTensor2D>(3, 1);
-                (*loss_val_tensor)(0, 0) = static_cast<float>(_last_loss);
-                (*loss_val_tensor)(1, 0) = _ae_red_ptr->_ema_event;
-                (*loss_val_tensor)(2, 0) = _ae_red_ptr->_ema_normal;
+                int num_of_samples = _aec_data_set->dimension(0);
+                loss_val_tensor = std::make_shared<fTensor2D>(1, 1);
+                (*loss_val_tensor)(0, 0) = static_cast<float>(_last_loss); 
+                (*loss_val_tensor)(1, 0) = _ae_red_ptr->_ema_event; // Mask the following lines to get reduction in data tranfers sizes, or Unmask to enable AEC stats
```

Comment:
Add TODO here that we should envelope it with if statement and pass model arg that controls it.